docs/user/guides/access/index.rst

```diff
@@ -0,0 +1,39 @@
+How-to guides: security and access
+==================================
+
+⏩️ :doc:`Setup Single Sign-On (SSO) with GitHub, GitLab, or Bitbucket </guides/setup-single-sign-on-github-gitlab-bitbucket>`
+    When using an :doc:`organization </commercial/organizations>` on  |com_brand|,
+    you can configure SSO for your users to authenticate to Read the Docs.
+
+⏩️ :doc:`Setup Single Sign-On (SSO) with Google Workspace </guides/setup-single-sign-on-google-email>`
+    When using an :doc:`organization </commercial/organizations>` on  |com_brand|,
+    you can configure SSO for your users to authenticate to Read the Docs.
+    **Separate version for Google Workspace**.
```

Comment:
Not sure what this means?

Reply:
```suggestion
    This guide is written for Google Workspace.
```

It's written so it's easy to spot that the second item isn't a duplicate, otherwise the two how-tos look very identical.